Jenkinsfile

```diff
@@ -144,36 +226,28 @@ String hw_distro(String size) {
                                                           def_val: 'centos7'))
 }
 
-String qb_inst_rpms_functional() {
-    if (quickbuild()) {
-        // TODO: these should be gotten from the Requires: of RPMs
-        return "spdk-tools"
-    }
-
-    return ""
+String functional_packages() {
+    String target = hw_distro_target()
+    return functional_packages(target)
 }
 
-String qb_inst_rpms_run_test() {
+String functional_packages(String distro) {
+    string pkgs = get_daos_packages(distro)
+    pkgs += " openmpi3 hwloc ndctl " +
+            "ior-hpc-cart-4-daos-0 " +
+            "romio-tests-cart-4-daos-0 hdf5-mpich2-tests-daos-0 " +
+            "testmpio-cart-4-daos-0 fio " +
+            "mpi4py-tests-cart-4-daos-0 MACSio"
```

Comment:
FWIW, I want this all to go away eventually.  These should al (most?) be daos-tests dependencies.